lib/registration.pm

```diff
@@ -249,8 +249,13 @@ sub yast_scc_registration {
     my $ret = wait_serial "yast-scc-done-\\d+-", $timeout;
     die "yast scc failed" unless (defined $ret && $ret =~ /yast-scc-done-0-/);
 
-    script_run 'zypper lr';
-    assert_screen 'scc-repos-listed';
+    if (get_var('ONLINE_MIGRATION')) {
```

Comment:
and instead of asserting this repo list add something like zypper lr|grep string, string will be made from VERSION or what we expect Pool/Updates